libs/core/src/lib/feed-list-item/components/item/feed-list-item.component.ts

```diff
@@ -0,0 +1,144 @@
+import {
+  Component,
+  ViewEncapsulation,
+  ChangeDetectionStrategy,
+  Input,
+  SimpleChanges,
+  ElementRef,
+  ContentChild,
+  OnInit,
+  OnChanges,
+  ChangeDetectorRef
+} from '@angular/core';
+
+import { CssClassBuilder, applyCssClass } from '../../../utils/public_api';
+import { FEED_LIST_ITEM_PREFIX } from '../../constants';
+import { FeedListActionComponent } from '../feed-list-action/feed-list-action.component';
+import { FeedListAvatarComponent } from '../feed-list-avatar/feed-list-avatar.component';
+import { FeedListFooterComponent } from '../feed-list-footer/feed-list-footer.component';
+
+@Component({
+  selector: 'fd-feed-list-item',
+  templateUrl: './feed-list-item.component.html',
+  styleUrls: ['./feed-list-item.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class FeedListItemComponent implements OnInit, OnChanges, CssClassBuilder {
+  /** User's custom classes */
+  @Input()
+  class: string;
+  /**
+   * Author of feed
+   */
+  @Input()
+  authorTitle: string;
+  /**
+   * Link to Author of feed
+   */
+  @Input()
+  authorLink: string | null = null;
```

Comment:
`null` type is not necessary, we don't have strict null checks

Reply:
Yep, updated